### DISCUSSION TITLE
Is there a way to change the languages Git Hub assigns to a repo

### DISCUSSION THREAD

**Discussion by DataJuggler** [community/community#49646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a few projects that Git Hub lists the wrong tags.

For example, http://github.com/DataJuggler/DataJuggler.Blazor.Components is a C# / Blazor library.

One CSS file is more lines of code than all the C#, so you list the project as CSS. C# and Blazor are not listed.

Another project is a Blender Python script, and you list it as PowerShell.

Are these tags editable? Thanks

**Comment by spenserblack** [comment]
There are a few ways to tell github-linguist, the tool GitHub uses to detect languages, to ignore some languages. For example, [fish-shell marks their completions as documentation so that the repo's primary language is C++ instead of Shell](https://github.com/fish-shell/fish-shell/commit/70f2899fcd6b6195446701eaffd033baff307001). You can find out more about github-linguist overrides [here](https://github.com/github/linguist/blob/40421516a3842120a2227032420e4ef5791b234d/docs/overrides.md).

**Comment by DataJuggler** [comment]
Thank you, I will look at the link provided. 

I discovered after I posted, that the problem is on the summary of the repo:

![image](https://user-images.githubusercontent.com/26331133/224161793-ab51d7f5-fb0a-439e-9c21-ae1b0704720f.png)

If I click on the repo, the breakdown is visible:

![image](https://user-images.githubusercontent.com/26331133/224161942-b0c7bd45-9772-4472-8cc2-7dbe74343412.png)

Thanks for the information.
